crates/ra_syntax/src/ast/extensions.rs

```diff
@@ -39,61 +37,45 @@ fn text_of_first_token(node: &SyntaxNode) -> &SmolStr {
 
 impl ast::Attr {
     pub fn is_inner(&self) -> bool {
-        let tt = match self.value() {
-            None => return false,
-            Some(tt) => tt,
-        };
-
-        let prev = match tt.syntax().prev_sibling() {
+        let prev = match self.syntax().prev_sibling() {
             None => return false,
             Some(prev) => prev,
         };
 
         prev.kind() == T![!]
     }
```

Comment:
Let's just remove this function: it's not used, and buggy: siblings include only nodes, not tokens, so `prev.kind() == T![!]` is (and was before this PR) always false